### PR TITLE
exercise

### PR DESCRIPTION
**Pull request by jvidesDev**
Good afternoon, I send you today's exercise, I await your response.

### CODE REVIEW

**Review by andresdtd — CHANGES_REQUESTED** [review]
Please put the else in the diagram.

**Comment by jvidesDev** [comment]
Hello, the due changes have already been made in the diagram, I await your response, thank you

**Review by juanhsd — CHANGES_REQUESTED** [review]
Good evening! you developed your code well, but you are making mistakes in your diagram
Remember that we use the 'screen' to use the msgbox

**Comment by jvidesDev** [comment]
Good evening, the due changes have already been made to the diagram

**Review by andresdtd — APPROVED** [review]
I see your changes @JuanCito16 , good job!

File: juan_vides/ejercicio.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    abo = 0
+    noabo = 0
+    abosup = 0
+    Total = 0
+    
+    While Total <= 3000000
+        dinero = Int(InputBox("Cuanto va a abonar?"))
+        If dinero > 0 Then
+            abo = abo + 1
+            Total = Total + dinero
```
Comment (LuisJrDev) [inline juan_vides/ejercicio.md:12]: The same recommendation that I made to a colleague, for the variables it is more recommended that everything be in lowercase

**Review by LuisJrDev — APPROVED** [review]
Good evening, good job just keep in mind the recommendation I gave you in the algorithm.

**Review by GoJhon — COMMENTED** [review]
Hello @JuanCito16, nice job. I have the same recomendation of @juanhsd.

**Review by GoJhon — APPROVED** [review]
Good job!

**Review by juanhsd — APPROVED** [review]

**Pull request closed without merging**